oid_validation/pulp/oid_validation/oid_validation.py

```diff
@@ -194,16 +194,17 @@ def _check_extensions(self, cert_pem, dest, log_func, repo_url_prefixes):
 
         valid = False
         for prefix in repo_url_prefixes:
-            # Extract the repo portion of the URL
-            repo_dest = dest[dest.find(prefix) + len(prefix):]
-            try:
-                valid = cert.check_path(repo_dest)
-            except AttributeError:
-                # not an entitlement certificate, so no entitlements
-                log_func('The provided client certificate is not an entitlement certificate.\n')
-            # if we have a valid url check, no need to continue
-            if valid:
-                break
+            if dest.find(prefix) > -1:
```

Comment:
For the sake of clarity you could do ``if prefix not in dest:``, but it's up to you. It just saves someone who can't remember the exact behavior of ``find`` a few seconds looking at the docs.